src/inventory.cpp

```diff
@@ -729,26 +729,21 @@ void InventoryList::moveItemSomewhere(u32 i, InventoryList *dest, u32 count)
 u32 InventoryList::moveItem(u32 i, InventoryList *dest, u32 dest_i,
 		u32 count, bool swap_if_needed, bool *did_swap)
 {
-	if(this == dest && i == dest_i)
+	if (this == dest && i == dest_i)
 		return count;
 
 	// Take item from source list
-	ItemStack item1;
-	if(count == 0)
-		item1 = changeItem(i, ItemStack());
-	else
-		item1 = takeItem(i, count);
+	ItemStack item1 = count == 0 ? changeItem(i, ItemStack()) : takeItem(i, count);
 
-	if(item1.empty())
+	if (item1.empty())
 		return 0;
 
 	// Try to add the item to destination list
 	u32 oldcount = item1.count;
 	item1 = dest->addItem(dest_i, item1);
 
 	// If something is returned, the item was not fully added
-	if(!item1.empty())
-	{
+	if (!item1.empty()) {
```

Comment:
This file has been touched only for code-style changes. Why though?

Reply:
Well, I had to read it to find out what counts as swapping, and decided to improve the code-style "en passant"